CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# How to contbitute
+We are glad you are reading this, because we need volunteer works to improve vt-optimizer.
+
+## Testing
+Please write tests for every functionality.
+
+## Submitting changes
+Please send a [GitHub Pull Request to the dev branch](https://github.com/ibesora/vt-optimizer/pull/new/master), so that @ibesora can merge to master when everything is documented and is ready to go after passing all tests.
```

Comment:
The link should be https://github.com/ibesora/vt-optimizer/pull/new/dev instead of https://github.com/ibesora/vt-optimizer/pull/new/master

Reply:
Yes, I modified this part as advised.